### PR TITLE
Activity log: Update today view

### LINKED ISSUE #17293: Activity Log: remove Rewind button from today's block
Hide the ‘rewind’ button for ‘today’ since it would rewind to the end of today and that is right now.

### Before

image

### After

image

### LINKED ISSUE #17299: Activity Log: expanded 'today' as default view
We should default to open 'today' and show its events, and also label it `— Today`.

Mock also shows no rewind button for the present day as per https://github.com/Automattic/wp-calypso/issues/17293

### Before

image

### After

image

### PR DESCRIPTION
**Pull request by sirreal**
- Expand current day by default.
- Add " — Today" to current day.
- Hide Rewind button on current day. Note, this removes the vertical separator before the caret which is normal `FoldableCard` behavior.
- Improve README on `FoldableCard.expanded` prop.

## Testing
* Visit https://calypso.live/stats/activity?branch=update/activitylog/today-view
* Confirm the changes I've described work well!

### Before

before

### After

after

Fixes #17293
Fixes #17299

### CODE REVIEW

File: client/my-sites/stats/activity-log-day/index.jsx [hunk 1]
```diff
@@ -97,15 +104,20 @@ class ActivityLogDay extends Component {
...
 			tsEndOfSiteDay,
 		} = this.props;
 
+		const formattedDate = applySiteOffset( moment.utc( tsEndOfSiteDay ) ).format( 'LL' );
+
 		return (
 			<div>
-				<div className="activity-log-day__day">{ applySiteOffset( moment.utc( tsEndOfSiteDay ) ).format( 'LL' ) }</div>
+				<div className="activity-log-day__day">
+					{ isToday ? translate( '%s — Today', { args: formattedDate } ) : formattedDate }
```
Comment (tyxla) [inline client/my-sites/stats/activity-log-day/index.jsx]: Could be useful for translators to describe in a comment what the argument contains.

File: client/my-sites/stats/activity-log-day/index.jsx [hunk 2]
```diff
@@ -152,6 +166,13 @@ class ActivityLogDay extends Component {
 	}
 }
 
-export default connect( null, {
-	recordTracksEvent: recordTracksEventAction,
-} )( localize( ActivityLogDay ) );
+export default connect(
+	( state, { tsEndOfSiteDay } ) => ( {
+		isToday:
+			Date.now() <= tsEndOfSiteDay &&
```
Comment (tyxla) [inline client/my-sites/stats/activity-log-day/index.jsx]: Nit: we're calling `Date.now()` twice, when we can call it just once?

Comment (sirreal) [inline client/my-sites/stats/activity-log-day/index.jsx]: 0e685a2

File: client/my-sites/stats/activity-log-day/index.jsx (hunk 1 above)
Comment (sirreal) [inline client/my-sites/stats/activity-log-day/index.jsx]: Added a `comment` in 949281c

> Long date with today indicator, i.e. "January 1, 2017 — Today"

Is that right?

Comment (tyxla) [inline client/my-sites/stats/activity-log-day/index.jsx]: Great, thanks! 👍

**Comment by tyxla** [comment]
:shipit:

**Pull request merged**